add panel

ocefpaf opened the pull request:
Ping @rsignell-usgs who requested the packaged.

@philippjfr I added you as a maintainer, hope is is OK.
Let me know if there are others I should add.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/panel```) and found it was in an excellent condition.

philippjfr commented:
Thanks so much @ocefpaf, meant to ping you earlier but was holding off until I fixed some issues with the pip packages (which @ceball now addressed).

ocefpaf commented:
> Thanks so much @ocefpaf, meant to ping you earlier but was holding off until I fixed some issues with the pip packages (which @ceball now addressed).

Yep. I was waiting for the PyPI package too but it seems that @rsignell-usgs needs it ASAP :wink:

rsignell-usgs commented:
@ocefpaf , thanks so much.  This will allow me to confidently add panel to my all conda-forge environment!
